### DISCUSSION TITLE
Importer missing banks supported by GoCardless

### DISCUSSION THREAD

**Discussion by digitlength** [firefly-iii/firefly-iii#8642] [Firefly III] [answered]
I have a bank I would like to import from, they are supported by GoCardless (https://gocardless.com/bank-account-data/coverage/), but their name does not appear in the drop down for my country in the Firefly Importer

Is the "Select your bank in..." list the only ones supported by Firefly, or can we use any that are supported by GoCardless?

**Comment by JC5** [comment, marked as answer]
The list is provided by GoCardless. I support everything they provide.

**Reply by digitlength** [reply]
OK, so just to be clear: if they say a bank is supported because it is listed at https://gocardless.com/bank-account-data/coverage/, that doesn't mean it is one that they provide in the list to Firefly?

**Reply by JC5** [reply]
Apparently not. I'm sorry, I never knew there was a difference. You could ask their support team about it.

**Reply by digitlength** [reply]
OK, I'll ask them - see if they respond. Does Firefly have some sort of special access to GoCardless that I need to reference?

**Reply by digitlength** [reply]
Actually, ignore that, I'm an idiot. It's listed with "The" in front of the bank name...

**Reply by JC5** [reply]
Hah! False alarm 😅
